components/new-homepage/GetToKnowUsSection.js

```diff
@@ -0,0 +1,274 @@
+          </H2>
+        </Box>
+        <Box width={['288px', '700px', '768px']}>
+          <P
+            fontSize={['18px', '20px']}
+            letterSpacing={[null, '-0.008em']}
+            lineHeight={['26px', '28px']}
+            textAlign="center"
+            color="black.800"
+            fontWeight="500"
+          >
+            <FormattedMessage
+              id="home.getToKnowUs.description"
+              defaultMessage={'Browse stories, blog posts, resources, and get support from the community.'}
+            />
+          </P>
+        </Box>
+      </Flex>
+      <Flex flexDirection={['column', 'row']} my="32px">
+        <Container
+          width={['288px', '342px', '470px', null, '536px']}
+          py="48px"
+          px="32px"
+          border="2px solid"
+          borderColor="primary.900"
+          borderRadius="16px"
+          mb={[3, 0]}
+        >
+          <H3 fontSize={['32px']} lineHeight={['40px']} letterSpacing={['-0.008em']} color="primary.900">
+            <FormattedMessage id="home.exploreAndLearn" defaultMessage={'Explore and learn'} />
+          </H3>
+          <P my="24px" fontSize="18px" lineHeight="26px" fontWeight="500" color="black.800">
+            <FormattedMessage
+              defaultMessage={'Check out these amazing Collectives—explore their transparent budgets and activities.'}
+            />
+          </P>
+          <ListContainer>
+            <StyledListItem fontSize="18px" lineHeight="26px" fontWeight="500" color="black.800" mb="24px">
+              <FormattedMessage
+                values={{
+                  link: getI18nLink({
+                    href: 'https://blog.opencollective.com',
+                    openInNewTab: true,
+                    color: 'black.800',
+                  }),
+                  u: I18nUnderline,
+                }}
+                defaultMessage={'<link><u>Read our blog</u></link>'}
+              />
+            </StyledListItem>
+            <StyledListItem fontSize="18px" lineHeight="26px" fontWeight="500" color="black.800" mb="24px">
+              <FormattedMessage
+                values={{
+                  link: getI18nLink({
+                    href: '/e2c',
+                    color: 'black.800',
+                  }),
+                  u: I18nUnderline,
+                }}
+                defaultMessage={'<link><u>Our exit strategy</u></link>'}
+              />
+            </StyledListItem>
+            <StyledListItem fontSize="18px" lineHeight="26px" fontWeight="500" color="black.800" mb="24px">
+              <FormattedMessage
+                values={{
+                  link: getI18nLink({
+                    href: '/become-a-sponsor',
+                    color: 'black.800',
+                  }),
+                  u: I18nUnderline,
+                }}
+                defaultMessage={'<link><u>More about Sponsors</u></link>'}
+              />
```

Comment:
`getI18nLink` returns a `StyledLink`. To make sure it uses the standard router, we must provide our `Link` component. It also supports underlining out of the box, so we can simplify the string.

```suggestion
              <FormattedMessage
                values={{
                  link: getI18nLink({
                    as: Link,
                    href: '/become-a-sponsor',
                    color: 'black.800',
                    textDecoration: 'underline'
                  }),
                }}
                defaultMessage={'<link>More about Sponsors</link>'}
              />
```

But here we should also simplify further, there's no benefit in having these styling components in the message and it will complexity the work of translators:

```suggestion
              <StyledLink as={Link} href="/become-a-sponsor" textDecoration="underline" color="black.800">
                <FormattedMessage defaultMessage="More about Sponsors" />
              </StyledLink>
```

Feel free to apply this change on similarly added strings.